experimental/bindings/java/build.gradle

```diff
@@ -54,8 +58,30 @@ android {
 
     externalNativeBuild {
         cmake {
-            version "3.13.0+"
+            version "3.18.4+"
             path "../../../CMakeLists.txt"
         }
     }
 }
+
+// Task to cmake configure the host
+task cmakeConfigureHost(type: Exec) {
+    doFirst {
+        println "Configuring host tools with cmake..."
+    }
+    workingDir cmakeListDir
+    commandLine "cmake", "-G" , "Ninja",  "-B", hostBuildDir ,  "-DCMAKE_C_COMPILER=clang", "-DCMAKE_CXX_COMPILER=clang++",  "-DCMAKE_INSTALL_PREFIX=${hostInstallDir}", "."
```

Comment:
Can you drop the requirement to use clang? We support other compilers (MSVC in particular, but also gcc).

Reply:
On my machine (linux) this doesn't compile unless you specify the compiler in this command. Is there another way to set it?

Reply:
Scratch that. Looks like it works after exporting environment variables: https://stackoverflow.com/questions/45933732/how-to-specify-a-compiler-in-cmake/45934279#45934279